contracts/rewards/StableLPMaker.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: MIT
+// P1 - P3: OK
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../interfaces/ISwap.sol";
+
+contract StableLpMaker is Ownable {
+    using SafeMath for uint256;
+    using SafeERC20 for IERC20;
+
+    ISwap public stableSwap;
+    address public pTri;
```

Comment:
can you put in comments what are the addresses pTRI, dao etc?

Reply:
done